<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock Data Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Stock Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch and Plot Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="chart_type_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Chart Type:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chart_type_radiobutton1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chart_type_radiobutton2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Candlestick</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="zoom_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Zoom</string>
    </property>
   </widget>
   <widget class="FigureCanvasTkAgg" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>109</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FigureCanvasTkAgg</class>
   <extends>QWidget</extends>
   <header>figurecanvastkagg.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
